model/schema.json

```diff
@@ -62,9 +82,21 @@
           "type": "string",
           "format": "uuid"
         },
+        "_version": {
+          "description": "The version of the document in the index",
+          "type": "integer"
+        },
         "agent": { "$ref": "#/definitions/agent-metadata" },
-        "host": { "$ref": "#/definitions/host" }
-      }
+        "host": { "$ref": "#/definitions/host-metadata" },
+        "server": { "ref":  "#/definitions/server-metadata" }
```

Comment:
Can the server metadata be used in two places? I think I need to read up on how this works. What is definition and hwat is actual usuage?

Reply:
The reason I named them `*-metadata` is because all the metadata objects are shared references from multiple objects in the definition.